src/ImageUploader/ImageUploader.tsx

```diff
@@ -11,7 +11,8 @@ export default function ImageUploader({
   titleVariant,
   subText = "A default image will be used if no image is uploaded",
   dropzoneText = "Drag 'n' drop an image file here, or click to select",
-  maxFileSize = 1000000,
+  isUploading = false,
+  maxFileSize = 1000000000,
```

Comment:
not sure about the default maxFileSize here

Reply:
I came to a conclution to set default maxFileSize to 1GB after a chat with @mattcorner to be consistent in VIRTO apps

Reply:
For reference, I said that this is what every other ImageUploader in VIRTO is set to and at least this would be consistent. I think we need to agree a better limit across the board.

Reply:
Most global websites have a 10MB limit https://support.buffer.com/article/617-ideal-image-sizes-and-formats-for-your-posts. I can't see a use case for exceeding this, certainly as a RUI default.

Reply:
Thanks @mattcorner , i think we should go with 10 MB for now.